docs/reference/analysis/tokenfilters/synonym-graph-tokenfilter.asciidoc

```diff
@@ -87,11 +87,17 @@ changes to synonym files. Only to be used for search analyzers.
 * `expand` (defaults to `true`).
 Expands definitions for equivalent synonym rules.
 See <<synonym-graph-tokenizer-expand-equivalent-synonyms,expand equivalent synonyms>>.
-* `lenient` (defaults to `false`).
-If `true` ignores errors while parsing the synonym configuration.
+* `lenient` (defaults to the value of the `updateable` setting).
+If `true` ignores errors while parsing the synonym rules.
 It is important to note that only those synonym rules which cannot get parsed are ignored.
 See <<synonym-graph-tokenizer-stop-token-filter,synonyms and stop token filters>> for an example of `lenient` behaviour for invalid synonym rules.
 
+[INFO]
```

Comment:
I like the extra information, but I'm thinking that this should be part of release notes and it will quickly get old (i.e. this will still be around in 9.x).

Maybe docs team has more guidance on this. I wouldn't include it as I don't see this kind of advice in other parts of our docs.

Reply:
I agree, this is what release notes and breaking changes are for. This callout should be removed.

Reply:
++, let's remove the version dependent callout.

If you feel that we should have more information in our documentation, you could add information on the `lenient` parameter that if lenient is set to true invalid synonyms will be ignored/skipped. That's potentially a little more clear than "errors are ignored" That might get at the underlying concern but still provide non-stale information to users.

It could be good to backport the warnings about lenient set to true to older branches.

Reply:
Ok, I will remove this callout. This information is already in the release notes per the changelog.

@kderusso Carlos already added a lot of info about the `lenient` parameter in #110822, which was backported all the way back to 8.10. So I think we're good there.